portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -10,9 +10,10 @@
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
-  }
+    private String first_name = "Tommy";
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        response.setContentType("text/html;");
+        response.getWriter().println("Hello " + first_name + "!");
```

Comment:
Why leave this as a variable if the value is always the same? If you put "Hello Tommy!" here I believe the behavior would be the same.

Reply:
Ok then for readability it would suggest leaving it as one string!

Reply:
Oh I see you did. Great :) thanks